components/atoms/a-images-grid.vue

```diff
@@ -52,10 +52,10 @@ export default {
   }
```

Comment:
In my opinion, we should update dummy images to this from SF UI example on product page, because now we have inconsistent spaces between images and this not looks so good.

Example on SF UI: https://prnt.sc/qya5uh
How it looks now on our demo: https://prnt.sc/qya6vm

Reply:
Good point. I've changed those dummy images as you suggested.

![Screenshot_2020-02-06 Home Page - Vue Storefront](https://user-images.githubusercontent.com/56868128/73918873-c093fa00-48c2-11ea-84d6-95ef93562908.jpg)

Reply:
Great